common/lcs/tests/serde.rs

```diff
@@ -186,18 +188,18 @@ proptest! {
 
         let v: BTreeMap<Vec<u8>, Vec<u8>> = v.into_iter().map(|(k, v)| {
             let mut k_bytes = Vec::with_capacity(k.len() + 4);
-            k_bytes.extend_from_slice(&(k.len() as u32).to_le_bytes());
+            k_bytes.extend_from_slice(&(k.len() as u8).to_le_bytes());
             k_bytes.extend(k.into_iter());
             let mut v_bytes = Vec::with_capacity(v.len() + 4);
-            v_bytes.extend_from_slice(&(v.len() as u32).to_le_bytes());
+            v_bytes.extend_from_slice(&(v.len() as u8).to_le_bytes());
```

Comment:
what if len is greater than 255?

Reply:
Oh right, I was supposed to revisit this and I forgot. Currently, proptest never generates large vectors in practice but who knows in the future...

Reply:
After some investigation, the right solution to use `prop_assume!`